src/main/java/Number.java

```diff
@@ -0,0 +1,19 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Number {
+
+    public String create() {
+        List<Integer> list = new ArrayList<>();
+        int i = 0;
+        while (i < 3) {
+            int number = (int)(Math.random() * 9 + 1);
+            if (!list.contains(number)) {
+                list.add(number);
+                i = i + 1;
+            }
+        }
+        return list.stream().map(item -> String.valueOf(item)).collect(Collectors.joining());
```

Comment:
stream이 대표적인 예입니다. 한 줄에 한 ``.`` 만 사용하는 것이 가독성이 좋습니다. 해당 코드의 경우 
```suggestion
        return list.stream()
                .map(item -> String.valueOf(item))
                .collect(Collectors.joining());
```
이렇게 사용하게 되면 가독성이 늘어납니다. 그리고 추가적으로 이렇게 한줄에 한 ``.``을 사용하게 되면,  
인텔리제이 IDE가 각 줄의 우측에 현재 파이프 라인에서 ``반환되는 타입``을 표기해주게 되고 
의도하는 대로 반환값이 나오는지, 실시간으로 확인할 수 있어 편리합니다.

Reply:
인텔리제이 IDE를 더 효율적으로 사용하는 방법인 것 같습니다. 가독성 면에서도 더 좋은 것 같습니다.  습관화해서 반영하도록 하겠습니다.